app/controllers/api/v1/users_controller.rb

```diff
@@ -29,10 +29,14 @@
 
 module Api
   module V1
-    class UsersController < ApplicationController
+    class UsersController < Api::V1::BaseController
```

Comment:
__[Reek]__

[InstanceVariableAssumption] Api::V1::UsersController assumes too much for instance variable '@user'


<!-- Comment by [SideCI](https://sideci.com) -->